src/components/light/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './style.css';
+import lightOff from './light-off.svg';
+import lightOn from './light-on.svg';
+import { useState } from 'react';
+
+export const Light = ({ name, state }) => {
+  const [lightState, setLightState] = useState(state);
+
+  const handleClick = () => {
+    if (lightState === 'on') {
+      setLightState('off');
+    } else {
+      setLightState('on');
+    }
+  };
```

Comment:
a právě tady kdybys to měla booleanem, tak se ti to píše o trochu líp :) 
```
  const [lightState, setLightState] = useState(state === "on");

  const handleClick = () => {
      setLightState(!lightState);
  };
```
ale i teĎ to jde zkrátit
```
  const handleClick = () => {
      setLightState(lightState === 'on' ? 'off' : "on");
  };
```